mate-panel/menu.c

```diff
@@ -265,28 +265,22 @@ static void
 add_drawers_from_alias (MateMenuTreeAlias *alias,
 			const char     *toplevel_id)
 {
-	MateMenuTreeItem *aliased_item;
-
-	aliased_item = matemenu_tree_alias_get_item (alias);
-
-	switch (matemenu_tree_item_get_type (aliased_item)) {
+	switch (matemenu_tree_alias_get_aliased_item_type (alias)) {
 	case MATEMENU_TREE_ITEM_DIRECTORY:
-		add_drawers_from_dir (MATEMENU_TREE_DIRECTORY (aliased_item),
+		add_drawers_from_dir (matemenu_tree_alias_get_directory (alias),
```

Comment:
Hmm, `matemenu_tree_alias_get_directory` returns result of `matemenu_tree_item_ref`. Should we unref it after using?